### ISSUE TITLE
NVDA quits reading Microsoft Teams aria-live regions

### ISSUE THREAD

**Issue by alexstinewaystar** [nvaccess/nvda#15102]
### Steps to reproduce:
1. Start NVDA.
2. Open Microsoft Teams.
3. Browse around channels/chats.
4. Eventually notice how message send notifications and the suggestions list (Control+G) is no longer read aloud.
5. This is because the aria-live region randomly quits being announced. I cannot figure out exact reproduction steps or conditions that this happens under. Not even sure if it is specific to NVDA as restarting NVDA does not help. Only restarting Teams helps.

### Actual behavior:

After some time, aria-live regions are not announced.

### Expected behavior:

The aria-live regions should always be announced when they are available.

#### NVDA installed/portable/running from source:
Installed.

#### NVDA version:
2023.1.0.27913.

#### Windows version:
Windows 10 Enterprise: 10.0.19044 Build 19044.

#### Name and version of other software in use when reproducing the issue:
Microsoft Teams: 1.6.00.16472.

#### Does the issue still occur after restarting your computer?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No.

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes.

NOTE: This has also been reported to the Enterprise Disability Answer Desk but this bug is dragging on way too long. Trying to find answers as restarting Teams multiple times a day at work is not a good solution.

**Comment by seanbudd** [comment]
Unfortunately this is an issue with Teams that we can do nothing about

**Comment by alexstinewaystar** [comment]
@seanbudd I figured it was a reach at best but thanks for confirmation. 👍
